src/zawrs.adoc

```diff
@@ -0,0 +1,115 @@
+== RISC-V Wait-on-Reservation-Set (Zawrs) extension
+
+=== Contributors
+
+This RISC-V specification has been contributed to directly or indirectly by:
+
+Aaron Durbin, Abel Bernabeu, Allen Baum, Christoph Müllner, David Weaver, Greg Favor, Josh Scheid, Ken Dockser, Paul Donahue, Phil McCoy, Philipp Tomsich, Tariq Kurd, Ved Shanbhogue
+
+=== Introduction
```

Comment:
@wmat - whats the convention we have been following with contributors. If this list gets folded into the top level contributors section then it can be dropped from this chapter. Once that is dropped, the "Introduction" heading can also be removed.

Reply:
I've been leaving the contributors section of each chapter as is.  My thinking is that each chapter may have a distinct set of contributors who would only prefer attribution for that chapter.  However, I'm also open to rolling all contributors into the overall contributors list.  I'm not sure which is best.  Another option would be expanding the overall contributors list but breaking it out by chapter.

Reply:
For other extensions integrated into the manual (such as Svpbmt, Svinval, Ztso) the manual has not had contributors per chapter. Having contributors per chapter would be new. Maybe something to discuss about maintaining the overall look and feel of the manual.

Reply:
Yes, I believe consistency across chapters is important.  And I'm leaning now toward there only being one main contributrs section.

Reply:
Agree.

Reply:
I have moved contributors to the top level contributors section and deleted the section and introduction title.

Reply:
Somehow that commit has not made it in. I still see the contributors section and the introduction title..

Reply:
Perhaps you looked right before I pushed?

Reply:
Looks good to me. Thanks!